client/material/tabs.tsx

```diff
@@ -139,5 +171,52 @@ export function Tabs<T>({ children, activeTab, onChange, className }: TabsProps<
     return tabElems
   }, [activeTab, children, onChange])
 
+  useKeyListener({
+    onKeyDown: (event: KeyboardEvent) => {
+      if ((event.keyCode === PAGEUP || event.keyCode === PAGEDOWN) && event.ctrlKey === true) {
+        const enabledChildren: React.ReactElement[] = []
+        React.Children.forEach(children, (child, i) => {
+          if (child && !(child.props as TabItemProps<T>).disabled) {
+            enabledChildren.push(child)
+          }
+        })
+
+        if (enabledChildren.length < 2) {
+          return true
+        }
+
+        let previousOrNextTab: T | undefined
+        React.Children.forEach(enabledChildren, (child, i) => {
```

Comment:
Just for future reference, this is not something you should be doing. `React.Children.forEach` is meant for looping over the children of an element (as given to you by React), you should never be using it to loop over arrays that you yourself created, even if they contain elements.